src/ngraph/pass/algebraic_simplification.cpp

```diff
@@ -241,8 +241,8 @@ bool ngraph::pass::AlgebraicSimplification::run_on_function(std::shared_ptr<ngra
         }
 
         const Node& node = *n;
-        auto eh = ops_to_const_values.find(TI(node));
-        if (eh == ops_to_const_values.end())
+        auto eh = ops_to_simplifiers.find(TI(node));
```

Comment:
Does this want to be `auto&`?

Reply:
`const Node&` -> `auto&`?

Reply:
No, `auto& eh = ...`
I'm not sure what the find is returning.

Reply:
@diyessi could you please approve this one when you have a moment?